src/auth/activation.ts

```diff
@@ -48,8 +51,74 @@ export async function initialize(
  */
 async function showManageConnectionsOnStartup() {
     if (!ExtensionUse.instance.isFirstUse()) {
+        getLogger().debug(
+            'firstStartup: This is not the users first use of the extension, skipping showing Add Connections page.'
```

Comment:
Avoid verbose messages in logs and UI. 
```suggestion
            'firstStartup: not first use, skipping showing Add Connections page.'
```